test/stampit-specs.js

```diff
@@ -93,70 +93,136 @@ test('stampit().methods(a, b)', function () {
     'Should mixIn objects when multiple methods are passed.');
 });
 
-module('Basics State');
+module('Basics Props');
 
-test('stampit({}, state)', function () {
+test('stampit({}, props)', function () {
   var obj = stampit({}, { foo: { bar: 'bar' } }).create();
 
   equal(obj.foo.bar, 'bar',
-    'Should set default state.');
+    'Should set default props.');
 });
 
-test('stampit().state()', function () {
-  var obj = stampit().state({
+test('stampit().props()', function () {
+  var obj = stampit().props({
     foo: { bar: 'bar' },
-    stateOverride: false,
+    propsOverride: false,
     func1: function(){}
-  }).state({
+  }).props({
     bar: 'bar',
-    stateOverride: true,
+    propsOverride: true,
     func2: function(){}
   }).create();
 
   equal(obj.foo.bar, 'bar',
-    'Should set default state.');
+    'Should set default props.');
   equal(obj.bar, 'bar',
-    'Should set let you add by chaining .state().');
-  ok(obj.stateOverride,
-    'Should set let you override by chaining .state().');
+    'Should set let you add by chaining .props().');
+  ok(obj.propsOverride,
+    'Should set let you override by chaining .props().');
   ok(obj.func1,
     'Should mix functions.');
   ok(obj.func2,
     'Should mix functions.');
 });
 
-test('stampit({}, state).state()', function () {
+test('stampit({}, props).props()', function () {
   var obj = stampit(null, {
     foo: { bar: 'bar' },
-    stateOverride: false,
+    propsOverride: false,
     func1: function(){}
-  }).state({
+  }).props({
     bar: 'bar',
-    stateOverride: true,
+    propsOverride: true,
     func2: function(){}
   }).create();
 
   equal(obj.foo.bar, 'bar',
-    'Should set default state.');
+    'Should set default props.');
   equal(obj.bar, 'bar',
-    'Should set let you add by chaining .state().');
-  ok(obj.stateOverride,
-    'Should set let you override by chaining .state().');
+    'Should set let you add by chaining .props().');
+  ok(obj.propsOverride,
+    'Should set let you override by chaining .props().');
   ok(obj.func1,
     'Should mix functions.');
   ok(obj.func2,
     'Should mix functions.');
 });
 
-test('stampit().state(a, b)', function () {
-  var obj = stampit().state({
+test('stampit().props(a, b)', function () {
+  var obj = stampit().props({
     a: 'a'
   }, {
     b: 'b'
   }).create();
 
   ok(obj.a && obj.b,
-    'Should mixIn objects when multiple methods are passed.');
+    'Should mixIn objects when multiple objects are passed.');
+});
+
+module('Basics refs');
+
+test('stampit({}, null, null, refs)', function () {
```

Comment:
Signature is `stampit(methods, refs, enclose, props)`. You have `props` and `refs` swapped in the tests.